code/ai/aigoals.cpp

```diff
@@ -603,7 +615,7 @@ void ai_goal_purge_all_invalid_goals(ai_goal *aigp)
 	ship_obj *sop;
 
 	// only purge goals if a new goal is one of the types in next statement
```

Comment:
Does this comment still make sense? Did it make sense even before this change?

Reply:
Well, yes.  For example, issuing an attack order causes any previously issued ignore orders to be removed.

Reply:
Oh, I see what you mean.  I'll tweak the comment.